src/postmate.js

```diff
@@ -49,27 +49,44 @@ export const resolveOrigin = (url) => {
   return a.origin || `${protocol}//${host}`
 }
 
+const messageTypes = {
+  handshake: 1,
+  'handshake-reply': 1,
+  call: 1,
+  emit: 1,
+  reply: 1,
+  request: 1,
+}
+
 /**
  * Ensures that a message is safe to interpret
- * @param  {Object} message       The postmate message being sent
- * @param  {String} allowedOrigin The whitelisted origin
+ * @param  {Object}          message       The postmate message being sent
+ * @param  {String|Boolean}  allowedOrigin The whitelisted origin
+ *                                         or false to skip origin check
  * @return {Boolean}
  */
 export const sanitize = (message, allowedOrigin) => {
-  if (message.origin !== allowedOrigin) return false
+  if (!supportsMessageChannel &&
+    typeof allowedOrigin === 'string' &&
+    message.origin !== allowedOrigin
+  ) return false
   if (typeof message.data !== 'object') return false
   if (!('postmate' in message.data)) return false
-  if (message.data.type !== messsageType) return false
-  if (!{
-    'handshake-reply': 1,
-    call: 1,
-    emit: 1,
-    reply: 1,
-    request: 1,
-  }[message.data.postmate]) return false
+  if (message.data.type !== messageType) return false
+  if (!messageTypes[message.data.postmate]) return false
   return true
 }
 
+/**
+ * Ensures that a message is safe to interpret
```

Comment:
This docstring doesn't seem to make sense. Looks like it was copied from `sanitize`, above.

Reply:
Thanks for noticing 

Fix in https://github.com/dollarshaveclub/postmate/pull/88